python/demos/meshtie_convergence.py

```diff
@@ -0,0 +1,326 @@
+# Copyright (C) 2022 Sarah Roggendorf
+#
+# SPDX-License-Identifier:    MIT
+
+import dolfinx.fem as _fem
+from dolfinx.common import Timer, timing, TimingType, list_timings
+from dolfinx.fem import Constant, Function, VectorFunctionSpace
+from dolfinx.graph import create_adjacencylist
+from dolfinx.io import XDMFFile
+import numpy as np
+import ufl
+from mpi4py import MPI
+from petsc4py import PETSc
+from dolfinx_contact.helpers import (epsilon, lame_parameters,
+                                     rigid_motions_nullspace, sigma_func)
+from dolfinx_contact.meshing import (create_split_box_2D, create_split_box_3D,
+                                     horizontal_sine, create_unsplit_box_2d, create_unsplit_box_3d)
+from dolfinx_contact.meshtie import nitsche_meshtie
+
+
+# manufactured solution 2D
```

Comment:
Move comments into function as docstring (same comment for all the following functions).

Can the functions be typed?